source/Irrlicht/CIrrDeviceLinux.cpp

```diff
@@ -820,7 +829,7 @@ bool CIrrDeviceLinux::run()
 		{
 			XEvent event;
 			XNextEvent(XDisplay, &event);
-			if (XFilterEvent(&event, XWindow))
+			if (acceptsIME() && XFilterEvent(&event, None))

```

Comment:
Okay that doesn't work. Whatever.

Reply:
When I tried it for the first time it only worked when I used `None` instead of `XWindow`. It could be something else I did at the time that broke it though. I have changed it back.

Reply:
Uh so did this work for you? If I change it to `XFilterEvent(&event, XWindow)` it completey breaks any text input.